site/data/supporters.yaml

```diff
@@ -73,17 +73,14 @@ bronzeV1:
 
   - name: 'Denim Group'
     link: https://www.denimgroup.com/
-    logo: /img/supporters/denimgroup.png
     notes: $3,000 in donations
 
   - name: 'HostedScan'
     link: https://hostedscan.com/
-    logo: /img/supporters/hostedscan.png
     notes: $1,500 donation
 
   - name: 'HiSolution AG'
     link: https://www.hisolutions.com/
-    logo: /img/supporters/hisolutions.png
```

Comment:
Did these exist at some point and should we be re-adding them?

Reply:
Bronze v1 do not use images, that's only for v2. (They did exist at some point.)

Reply:
They got demoted in the sponsors review - which should be mentioned 😉
Re the updated webpages - another script could help...

Reply:
Okay

Reply:
We could add a zapbot monthly action that PRs the base blog posts (filling those automated parts).